src/components/JobApplications/JobApplications.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { fetchApplicationsData } from '../../apiCalls';
+import ClipLoader from "react-spinners/ClipLoader";
+import { useUserLoggedContext } from '../../context/UserLoggedContext';
+import useSWR from 'swr';
+
+interface JobApplication {
+  id: string;
+  position_title: string;
+  date_applied: string;
+  status: number;
+  notes: string;
+  job_description: string;
+  application_url: string;
+  contact_information: string;
+  company_id: number;
+  company_name?: string;
+}
+
+const statusMap: { [key: number]: string } = {
+  1: 'Submitted',
+  2: 'Interviewing',
+  3: 'Offer',
+  4: 'Rejected',
+  5: 'Phone Screen',
+};
+
+const statusStyles: { [key: string]: string } = {
```

Comment:
You gotta gimme the skinny on this guy!! Are you using a variable to create some styling to apply to a piece of code below??   |
                                                          V